retreaver-typescript/src/base/Request.ts

```
@@ -0,0 +1,265 @@
+  }
+
+  /**
+   * Make API request with cross-browser support
+   * @param requestUri - The request URI
+   * @param callbackFunctions - Callback functions
+   * @param payload - Post object
+   */
+  apiRequest(requestUri: string, callbackFunctions: Function | Function[], payload?: any): void {
+    const httpPrefix = this.config.http_prefix;
+    const addr = this.config.addr;
+    const urlregex = eval(this.config.urlregex);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)